edb/edgeql/ast.py

```diff
@@ -184,6 +184,10 @@ class AnyTuple(PseudoObjectRef):
     pass
 
 
+class AnyObject(PseudoObjectRef):
```

Comment:
Maybe we should merge `anytype/anyobject/anytuple` into one node for these things?

Reply:
I've merged the AST nodes and this cleans the code up a bit already.

Something similar could probably be also done on `s_types.Type` methods (`is_any`, `is_anyobject`, `is_anytuple` & friends).